testing/confd/APKBUILD

```diff
@@ -1,37 +1,37 @@
 # Contributor: tang0th <tang0th@gmx.com>
+# Contributor: fchavant <francois@lightcurve.io>
 # Maintainer:
 pkgname=confd
-pkgver=0.12.0_alpha3
-_ver=${pkgver/_/-}
+pkgver=0.16.0
 pkgrel=0
 pkgdesc="Manage local application configuration files using templates and data from etcd or consul"
 url="http://confd.io"
 arch="all"
 license="BSD"
```

Comment:
looks it needs update as well

Reply:
Actually it is MIT https://github.com/kelseyhightower/confd/blob/master/LICENSE
The list is https://spdx.org/licenses/